tests/platform/test_reload_config.py

```diff
@@ -0,0 +1,64 @@
+"""
+Check platform status after config is reloaded
+
+This script is to cover the test case 'Reload configuration' in the SONiC platform test plan:
+https://github.com/Azure/SONiC/blob/master/doc/pmon/sonic_platform_test_plan.md
+"""
+import logging
+import re
+import os
+import time
+import sys
+
+from ansible_host import ansible_host
+from utilities import wait_until
+from check_critical_services import check_critical_services
+from check_interface_status import check_interface_status
+from check_transceiver_status import check_transceiver_basic
+from check_transceiver_status import all_transceivers_detected
+
+
+def test_reload_configuration(localhost, ansible_adhoc, testbed):
+    """
+    @summary: This test case is to reload the configuration and check platform status
+    """
+    hostname = testbed['dut']
+    ans_host = ansible_host(ansible_adhoc, hostname)
+    ans_host.command("show platform summary")
+    lab_conn_graph_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), \
+        "../../ansible/files/lab_connection_graph.xml")
+    conn_graph_facts = localhost.conn_graph_facts(host=hostname, filename=lab_conn_graph_file).\
+        contacted['localhost']['ansible_facts']
+    interfaces = conn_graph_facts["device_conn"]
+    asic_type = ans_host.shell("show platform summary | awk '/ASIC: / {print$2}'")["stdout"].strip()
```

Comment:
line 27 to 33 are duplicated in several file, it is better to have a function for that.

Reply:
@lguohan Yes, these lines are duplicated in many places and should be improved.  But how about keep them for now and improve a little bit later?

The reason is that I plan to create PR to improve some of the design of using pytest in sonic-mgmt. The prelimary design improvement is shown here: https://github.com/wangxin/sonic-mgmt/pull/10.

When the new design is accepted, then I will update the platform testing scripts using the new design to have better code reuse. By this approach, we can same some effort.

What do you think?